{{cookiecutter.directory_name}}/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: "3"
+
+services:
+  db:
+    image: postgres:14-alpine
+    volumes:
+      - ./data/db:/var/lib/postgresql/data
+    environment:
+      POSTGRES_DB: DATABASE_NAME
+      POSTGRES_USER: DATABASE_USER
+      POSTGRES_PASSWORD: DATABASE_PASSWORD
+    ports:
+      - "5432:5432"
+  django:
+    build: .
+    volumes:
+      - media_storage:/usr/local/app/media
+      - private_storage:/usr/local/app/private
+      - ./logs:/var/log
+    environment:
+      DATABASE_HOST: db
+      DATABASE_NAME: {{cookiecutter.description}}
+      DATABASE_USER: postgres
+      DATABASE_PASSWORD: postgres
+      DJANGO_SETTINGS_MODULE: {{cookiecutter.description}}.settings.development
+      REDIS_HOST: redis
+      SECRET_KEY: 'oqjwvmob^(qwlil^8ub8%a@o5@a!^x0j1*^*1m@y46k%(6+w'
+      SECURED_VIEW_JWK: '{  "p": "w0ZHQN8tDhB80-X0AoOnCx72mP0leUs7pRpyQ_fJVahS-6Rz34wMoyxsdjVFTOO1lp4y14l5MYBZ0GTpEEL5wOgLZXMPZmyjrUa3SVN-K0VKD8fDKdHVALYtqiHulT9OAIDzzHN1d1uXRnrlNoDLWkqaW-4yZC1bKTGJfV8yk3k",  "kty": "RSA",  "q": "q8WYI8dY5eTj12Q0l3KwWDOyVbgCX1bZUd7CEfoUoiRSOnk2HYnxx7aGri4Zh_vfJ1NM8lEkjHe91qIONbzqt-JwzXGV21UEQE6WtmKaMBHBaHz22NczkpiVcEL6otfjWNdsh3YxLpBnKNEndT4NhdnNL9vN4HNeqz8Uq_SVr5M",  "d": "DnPkzyaxOY1mg7a9Ndi2jqDJo4jo2zOPnSbf-W2mLWQlk_sZ1sf1OpwDeskUMVfW4zJ9RP3mdiCUoz9cT9tX6dZzVmJKo-n9n3RLSDo26lR_TV9ynUtq8A3dghzkoO-xe3kD7OFAWj5RrSr50Y9r_DkXRkuaXeN6rRN7_x5wX4Foh0GKfSQQEa2a5qxqyQGF3EYlyT_lhqKrtHpbtXVHo9RQJw2vX4gpyzdLBwNdrkc03dXKU2mZeRHLpBNQiejx8lS63OthzG5gjdBsIT3YtxaNtRHlvZQnKCau_nE4jT_jLl3IX2I3ZeRGrxyd4dYMbu84tgoDHRfWmpn92150sQ",  "e": "AQAB",  "qi": "GMsGGp-Ikcw9infNMCLwqRSXgZdyicLXZiFagG3vJcG2_1vfgFNAR0yIr855SJv-sCmhGK1lTGYZzi0rcPc3wLUT8Kd_G_SbYUenbHhioDDozvfTGlyw1v8PkNhjp7iI3oF2ANXWzUruK-Lp-WupbxYvqyUBUKpqp1o6XN4IW1k",  "dp": "e1-lmQjn8GDeXQqGFiiTm2UerB4UVQQBVRjWZLbiz2mftDl8tBpJybZBeJXDiQa9LcIZoS_MEEhldlKrJYkLPqPcv2Xoru-ooctx9bP4nOIUHtA5L-zQxLhVNY5mrIjogUz3u46fAqAtfWwb8UmCMTO9PqPJ1p0JYYdYtIs26xE",  "dq": "jImaNIvBXDatg2jk7pRTW5bQbfmjV0XDgIx01Ch8AT0eWeEaOYjHXNMl1hS5ptyYc6Bf8FIbSSIfHCBy59ytXc3S5QPtzAn-9G5YONjrtjCQwWCcxkUsWOCmQ7dJQTrJM8LtdNdq3eIfxPF7Cx2-Cqlsi5-XDfDWn914mEgAwCk",  "n": "gwaqui74iVp7fSiCe2A7L-P2YQ0CqO81LhbqqCCQ4Czd-QMlNv0z5mexMvqd0J5f71V3yJW02tFjZeIGat0uU5ZelQEiUojRshawupu3Ndedwoy7AaphOaUvjwwH_02awqWj7DXfDE_jgoULvZv6UkRUQtHhYp6ZW4xO_f8z8uUj2lUYv9PeaxQgexxLQRvjuRMIHN5tYuB1nYXKw_i1TDyHuGCXvefM2iKRk7kWlk7Far2pGqzuHnQ_OjTy9tgJbVZ0m2uYOmuVe48D17yQoleZr1dFv8zcIJ0KEsK5nMAdHeiEQdmZfukZ5RRR4i_S5a0Mj0bQ5vKxODnjZkdlew"}'
```

Comment:
Remove `SECURED_VIEW_JWK` if we are not going to use a JWT tokens. I would like also to open discussion on the JWT usage @zurek11.

Reply:
JWT is not going to be implemented in this MR